docs/docs/building-modules/collections/01-intro.md

```diff
@@ -0,0 +1,232 @@
+# Collections
```

Comment:
IMO, this should be collapsed in one README.md under the module itself.
Then we can put in under building-modules at docs build time.

Reply:
See https://github.com/cosmos/cosmos-sdk/blob/main/docs/pre.sh / https://github.com/cosmos/cosmos-sdk/blob/main/docs/post.sh